### DISCUSSION TITLE
Why is the source generator located in "analyzers/cs" instead of "analyzers/dotnet/cs" according to the Incremental Generators Cookbook?

### DISCUSSION THREAD

**Discussion by palapapa** [microsoft/CsWin32#1264] [Q&A]
I am trying to understand how CsWn32 is implemented. I came across the [Incremental Generators Cookbook](https://github.com/dotnet/roslyn/blob/main/docs/features/incremental-generators.cookbook.md) which I think is what this project uses under the hood. It says that if you want to publish your generator as a nuget package, you must put the generator DLL in the `analyzers/dotnet/cs` folder in the package, but somehow this package puts it under `analyzers/cs` and it still works?

**Comment by AArnott** [comment]
This generator doesn't follow that cookbook and is not an incremental generator. We may switch to that eventually.

As to the `\dotnet\` part of the path... I believe that's simply optional. I don't know what purpose it serves for sure.
